### PR TITLE
[TACHYON-1428] Documented some of the Under FS packages in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: common/src/test/java/tachyon/master/block/BlockIdTest.java [hunk 1]
```diff
@@ -18,10 +18,17 @@
...
+/**
+ * Tests for the {@link BlockId} class.
+ */
 public final class BlockIdTest {
 
   private static final long MAX_SEQUENCE_NUMBER = 16777215L;
 
+  /**
+   * Tests that the {@link BlockId#createBlockId(long, long)} method works correctly when creating a
+   * block with the {@link BlockId#getMaxSequenceNumber()} method.
```
Comment (aaudiber) [inline common/src/test/java/tachyon/master/block/BlockIdTest.java]: works correctly when creating a block with the maximum sequence number

File: underfs/glusterfs/src/main/java/tachyon/underfs/glusterfs/GlusterFSUnderFileSystem.java [hunk 2]
```diff
@@ -36,6 +36,13 @@
    */
   public static final String SCHEME = "glusterfs://";
 
+  /**
+   * Constructs a new Gluster FS {@link tachyon.underfs.UnderFileSystem}.
+   *
+   * @param fsDefaultName the under FS prefix
+   * @param tachyonConf the configuration for Tachyon
+   * @param conf the configuration
```
Comment (aaudiber) [inline underfs/glusterfs/src/main/java/tachyon/underfs/glusterfs/GlusterFSUnderFileSystem.java]: conf the configuration for Hadoop or GlusterFS

File: underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java [hunk 3]
```diff
@@ -55,6 +55,13 @@
   private static final FsPermission PERMISSION = new FsPermission((short) 0777)
       .applyUMask(FsPermission.createImmutable((short) 0000));
 
+  /**
+   * Constructs a new HDFS {@link UnderFileSystem}.
+   *
+   * @param fsDefaultName the under FS prefix
+   * @param tachyonConf the configuration for Tachyon
+   * @param conf the configuration
```
Comment (aaudiber) [inline underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java]: the configuration for Hadoop

**Comment by aaudiber** [comment]
Thanks for the doc improvements! I've left a few comments

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. I updated the PR accordingly.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**